PullRequestCreated/PullRequestCreated.ts

```diff
@@ -5,16 +5,24 @@ import { JIRA_ISSUE_PATTERN, JIRA_TASK_ISSUE } from '../lib/Constants';
 class PullRequestCreated extends OctokitAction {
   protected async execute(): Promise<void> {
     const pr = await this.getPullRequest(this.payload.pull_request.number);
-    if (this.shouldCreateIssue(pr)) {
+    const linkedIssues = pr?.title?.match(JIRA_ISSUE_PATTERN) || null;
```

Comment:
The null-check for PR is here 3 times now. That's where it makes sense to either do an early return, or nest the `if`